index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <script defer src="script.js"></script>
+    <title>Survey Form</title>
+</head>
+<body>
+    
+    
+        
+    <header class="header">
```

Comment:
Kindly  I recommend that you take away the spacing, this will make your code clean and readable.

```
<body>
<header class="header">
```